elastic/kibana
# [Alerting] Fixing `es_search_duration_ms` when search result is empty

## Summary

The ES client returns an empty object when searching over missing indices so the `took` value comes back undefined. Since we are accumulating a total value for `took` to use for `esSearchDuration`, this meant `esSearchDuration` was being set to `0 + undefined = NaN` which couldn't be indexed by the event log.

This PR defaults to using `0` in this scenario where a `took` value is unavailable.

## To Verify

Create a `Missing monitoring data` rule on a fresh ES/Kibana install where you have no metrics or monitoring indices. Verify that the event log `execute` event is indexed correctly and that `kibana.alert.rule.execution.metrics.es_search_duration_ms` is set to `0`.

### Checklist
